webapp/src/components/call_widget/component.tsx

```diff
@@ -1183,7 +1184,8 @@ export default class CallWidget extends React.PureComponent<Props, State> {
     }
 
     renderRecordingDisclaimer = () => {
-        if (this.state.recDisclaimerDismissedAt) {
+        if (!this.props.callRecordingStartAt ||
+            this.state.recDisclaimerDismissedAt > this.props.callRecordingStartAt) {
```

Comment:
Should this check be used before calling the `renderRecordingDisclaimer` fn?

Reply:
That's a question I asked myself as well. I don't know what's the best practice but I was trying to avoid making the code there even heavier so opted to keep everything inside.